the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js

```diff
@@ -83,6 +85,11 @@ const JoinRoom = () => {
       handleKick(currentName);
     }
   };
+  const handleSelectChange = (event) => {
+    let value = event.target.value;
+    const response = axios.post(ip + `user/play/${value}/${id_room}`);
+
+  };
```

Comment:
Add space before and after a function to improve readability.
Remove excessive empty line inside the function.